soup_test.py

```diff
@@ -0,0 +1,46 @@
+from bs4 import BeautifulSoup
+from urllib.request import urlopen
+import urllib.error
+
+url = "https://lpi.oregonstate.edu/mic/glossary" #specify source location
```

Comment:
@hawkicad in the future we could turn this into an environment variable instead for easier access/changes after hand-off?

Reply:
Sounds good